Player/Console/PlayerCommand.php

```diff
@@ -60,6 +61,8 @@ protected function configure()
                 new InputOption('sandbox', '', InputOption::VALUE_NONE, 'Enable the sandbox mode', null),
                 new InputOption('ssl-no-verify', '', InputOption::VALUE_NONE, 'Disable SSL certificate verification', null),
                 new InputOption('blackfire-env', '', InputOption::VALUE_REQUIRED, 'The blackfire environment to use'),
+                new InputOption('junit', '', InputOption::VALUE_REQUIRED, 'Save output execution report as JUnit to file'),
+
```

Comment:
extra line

Reply:
I think the description should mention that the value is the path to the file where the the output will be written.